webauthn-rs-core/src/internals.rs

```diff
@@ -1050,16 +1050,16 @@ fn tpmtsignature_parser(input: &[u8]) -> nom::IResult<&[u8], TpmtSignature> {
 /// [1]: https://trustedcomputinggroup.org/wp-content/uploads/TCG-TPM-VendorIDRegistry-v1p06-r0p91-pub.pdf,
 #[derive(Debug)]
 pub enum TpmVendor {
-    AMD,
+    Amd,
```

Comment:
Changes here are going to have downstream implications... will need to make it clear when bumping the version

Reply:
This was added this cycle, so that's okay.

Saying that, these are acronyms of vendors, so they should remain as they were capitalised, and clippy should be told to ignore this :)

Reply:
💯, ❤️